xml/ha_storage_protection.xml

```diff
@@ -655,6 +664,16 @@ Timeout (msgwait)  : 10
     <para> If the SBD device is not accessible, the daemon will fail to start and inhibit
      &corosync; start-up. </para>
    </step>
+   <step xml:id="st.ha.storage.protect.sbd_delay_start">
+    <para>Search for the following parameter: <parameter>SBD_DELAY_START</parameter>.</para>
+    <para>
+      Enables or disables a delay. Set it to <literal>yes</literal>, if you
+      want to delay the start of SBD on boot. This parameter can be
+      necessary if your cluster nodes boots very fast whereas the other
+      nodes are still waiting to return from the fencing action.
+      This is an occasional issue with virtual machines.
```

Comment:
Something like "This can occur more often with virtual machines."?

Reply:
Fixed with the above rephrase. :wink: